From ace4b4626bfb0bfc2b0b4afce210c086afa86bbf Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Mon, 4 Apr 2011 22:46:28 -0500
Subject: Modify the output of (version) to indicate Debian modifications.

The output of (version) has been modified to indicate Debian modifications.

Author: Rob Browning <rlb@defaultvalue.org>
Added-by: Rob Browning <rlb@defaultvalue.org>
---
 lisp/version.el |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/lisp/version.el b/lisp/version.el
index ebd4fea..b8107e6 100644
--- a/lisp/version.el
+++ b/lisp/version.el
@@ -56,8 +56,8 @@ to the system configuration; look at `system-configuration' instead."
   (interactive "P")
   (let ((version-string
          (format (if (not (called-interactively-p 'interactive))
-		     "GNU Emacs %s (%s%s%s)\n of %s on %s"
-		   "GNU Emacs %s (%s%s%s) of %s on %s")
+		     "GNU Emacs %s (%s%s%s)\n of %s on %s, modified by Debian"
+		   "GNU Emacs %s (%s%s%s) of %s on %s, modified by Debian")
                  emacs-version
 		 system-configuration
 		 (cond ((featurep 'motif)
